lib/processes/visTestPattern.cpp

```diff
@@ -463,23 +466,31 @@ void visTestPattern::compute_expected_data() {
     size_t num_freqs = freqs.size();
     size_t num_prods = prods.size();
 
+    INFO("Computing expected data...");
     expected.resize(num_freqs);
+    for (size_t f = 0; f < num_freqs; f++)
+        expected[f].resize(num_prods);
 
-    // Sort input feed values by frequency.
-    for (size_t f = 0; f < num_freqs; f++) {
+        // Sort input feed values by frequency.
+#ifdef _OPENMP
```

Comment:
One concern I have here is that OPENMP doesn't respects the cpu affinity model that kotekan uses.  On the `recv2` style nodes where having exclusive cores isn't as important, I think this is ok for these kind of testing `stages`.  However we might want to consider implementing some kind of control over the OPENMP thread affinity.  From what I can tell online I'm not sure that can actually be done on a per `stage`/MP call basis, but we might be able to set the global OPENMP affinity via the config file (then having kotekan internally modifying the OPENMP environment variables).   That is likely a change external to this PR, and as long as we don't think this will cause problems for say the networking threads or disk writing threads while the test is running, then I'm ok with change as is.